app/views/layouts/application.html.haml

```diff
@@ -35,6 +35,9 @@
     = csrf_meta_tags
     %meta{ name: 'style-nonce', content: request.content_security_policy_nonce }
 
+    %script{ src: '/MathJax/config.js' }
+    %script{ src: '/MathJax/es5/tex-chtml.js' }
```

Comment:
TODO: Should actually include MathJax in the yarn package.json or otherwise actually vendor it or whatevs instead of having it implicitly just on the server at a root domain